src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -61,6 +63,27 @@ internal static ActivityContext GetParentContext()
             return activityContext;
         }
 
+        internal static ActivityContext ExtractParentContext<TInput>(TInput input)
```

Comment:
I think this is an extremely useful feature and should be mentioned in the PR description, and the XML docs of the AWSLambdaWrapper class.

Reply:
Added to the PR description, and mentioned in the XML docs of the AWSLambdaWrapper class.